geotrek/feedback/parsers.py

```diff
@@ -32,7 +32,7 @@ def __init__(self):
         if settings.PAPERCLIP_ENABLE_LINK is False:
             raise Exception('You need to enable PAPERCLIP_ENABLE_LINK to use this function')
         try:
-            self.filetype = FileType.objects.get_or_create(type="Photographie", structure=None)
+            self.filetype, created = FileType.objects.get_or_create(type="Photographie", structure=None)
         except FileType.DoesNotExist:
```

Comment:
First time reviewing, hi 👋
This is not 100% related to your change, but do you think that exception handling is really mandatory ?